pyproject.toml

```diff
@@ -74,3 +74,14 @@ license-files = ["LICENSE", "COPYRIGHT"]
 include = ["dissect.*"]
 
 [tool.setuptools_scm]
+
+[tool.setuptools.package-data]
+"dissect" = ["pyproject.toml"]
+
+[project.scripts]
+# TODO: pick one :)
+dissect-update = "dissect.update:main"
```

Comment:
I think this is the best one. 

There's a long outstanding ticket (can't find it neither publicly nor internally atm) to add alternative names to _all_ of the Dissect tools with a `dissect-` prefix. For example, `dissect.util` has a `dump-nskeyedarchiver` but it's hard to discover it. This way you could do `dissect-<TAB>` and discover _all_ tools across all Dissect projects. `dissect-update` would fit right in there.